src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1163,6 +1163,11 @@
     "create index fact_values_value_float_idx on fact_values(value_float)"
     "create index fact_values_value_integer_idx on fact_values(value_integer)"))
 
+(defn add-corrective-change-index
+  []
+  (jdbc/do-commands
+    "CREATE INDEX resource_events_status_for_corrective_change_idx ON resource_events (status) WHERE corrective_change;"))
```

Comment:
In https://github.com/puppetlabs/puppetdb/pull/2078 I was envisioning this as an index on corrective_change where corrective_change=true. Part of the idea there was to benefit consumers other than the console. Have we checked to see if this index gives any benefit over that one?